2-mandatory/1-writers.js

```diff
@@ -71,7 +72,9 @@ Exercise 2:
 */
 
 function logDeadWritersInTheirForties() {
-  // write your code here
+for (let age in writers) {
+filter.writers (if ${writers.age} >=40 && <=55)   
```

Comment:
Did this statement give the correct answer? I am wondering if you do not need to reenter `${writers.age}` for the second condition.